deploy/roles/apache/tasks/main.yml

```diff
@@ -51,3 +49,20 @@
 - name: Save rules
   command: "/sbin/service iptables save"
   when: iptables_config
+
+- name: create certs for apache localhost
+  command: openssl req -x509 -nodes -days 365 -newkey rsa:2048 -subj "/C=US/ST=CFPB/L=Washington/O=Dis/CN=www.cf.gov" -keyout /etc/pki/tls/private/localhost.key -out /etc/pki/tls/certs/localhost.crt
+  args:
+    creates: /etc/pki/tls/certs/localhost.crt
+  when: not custom_repo
+
+- name: Allow admins to read logs
+  acl:
+    path: "{{ item }}"
+    entity: "{{ restricted_group_name }}"
+    etype: group
+    permissions: rx
+    state: present
+  with_items:
+    - /var/log/httpd24/
+    - /var/log/httpd/
```

Comment:
@stirlingv This should actually be controlled by a variable - Rather than doing both httpd and httpd24, this depends on which one is installed.  If we are using httpd24 (which is the current default), then /var/log/httpd won't exist.  So rather than doing "with_items" for both, just replace `{{ item }}`  with a new variable `{{ apache_log_folder }}` with a default of  `/var/log/httpd24`.  Also, we should have `recursive: yes` and `default: yes` to ensure that all files currently in that folder are readable by the admin user, and all new files ever created will also be readable.